x/group/keeper/invariants_test.go

```diff
@@ -0,0 +1,593 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	gogotypes "github.com/gogo/protobuf/types"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/store"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/cosmos/cosmos-sdk/x/group/internal/orm"
+	"github.com/cosmos/cosmos-sdk/x/group/keeper"
+)
+
+type invariantTestSuite struct {
+	suite.Suite
+}
+
+func TestInvariantTestSuite(t *testing.T) {
+	suite.Run(t, new(invariantTestSuite))
+}
+
+func (s *invariantTestSuite) SetupSuite() {
+	s.T().Parallel()
```

Comment:
let's just use `simapp.Setup` here as in the other keeper testsuite instead of building the ctx and db everytime